app/assets/stylesheets/customizations.css.scss

```diff
@@ -36,6 +36,10 @@
           float: left;
           width: 12rem;
           line-height: 1em;
+          /* Smartphones (portrait and landscape) ----------- */
```

Comment:
Source: http://responsivedesign.is/develop/browser-feature-support/media-queries-for-common-device-breakpoints